src/Microsoft.Diagnostics.Runtime/src/DataTargets/ClrInfo.cs

```diff
@@ -33,21 +32,15 @@ internal ClrInfo(DataTarget dt, ClrFlavor flavor, ModuleInfo module, DacInfo dac
         /// </summary>
         public ClrFlavor Flavor { get; }
 
-        /// <summary>
-        /// Returns module information about the Dac needed create a ClrRuntime instance for this runtime.
-        /// </summary>
-        public DacInfo DacInfo { get; }
```

Comment:
You've gotten rid of "DacInfo" here, but tracking information about the dac needed to debug the runtime is 100% something that I want to expose from this library.  For most external users, this is just an implementation detail to get ClrMD functional.

For internal users (myself included), I actually need a property that exposes the filename, timestamp, and filesize that the dac is expected to be indexed under.

In retrospect I shouldn't have made DacInfo inherit from ModuleInfo, but we do need to keep this class.

Reply:
thanks for the details, it was intended to inline DacInfo into ClrInfo, but I'll restore this as requested